jenkins-x.yml

```diff
@@ -102,6 +102,16 @@ pipelineConfig:
                   - -f
                   - /builder/home/basic-auth-pass
 
+              - name: tag-release
+                image: jenkinsxio/jx:2.0.128
+                command: git
+                args: ['tag', '-a', '${inputs.params.version}', '-m', '${inputs.params.version}']
+
+              - name: push-tag
+                image: jenkinsxio/jx:2.0.128
+                command: git
+                args: ['push', 'origin', '${inputs.params.version}']
+
```

Comment:
We shouldn't need to do the git tag here as this already happens in the pipelinerunner